pkg/kotsadm/api_objects.go

```diff
@@ -376,6 +377,16 @@ func apiDeployment(deployOptions types.DeployOptions) *appsv1.Deployment {
 								},
 							},
 							Env: env,
+							Resources: corev1.ResourceRequirements{
+								Limits: corev1.ResourceList{
+									"cpu":    resource.MustParse("1000m"),
+									"memory": resource.MustParse("1000Mi"),
+								},
+								Requests: corev1.ResourceList{
+									"cpu":    resource.MustParse("250m"),
```

Comment:
this is 200m in the kurl pr https://github.com/replicatedhq/kURL/pull/651/files#diff-283e21b0b927bd7b3cac539c357fab79R162

Reply:
I'll fix it